.gitignore

```diff
@@ -7,4 +7,4 @@ env/*
 *.pyc
 _pytest/*
 bin/
-.idea/
\ No newline at end of file
+.DS_Store
```

Comment:
This belongs in a global gitignore rather than here

Reply:
will do.